README.md

```diff
@@ -57,6 +63,8 @@ def duplicates?(array)
 end
 ```
 
+* SNIPPET 4: 0(n^2) quadratic (it's that or 0(n*m) but I admittedly don't understand that one)
```

Comment:
This one is in fact O(n * m). The reason is because the loops each depend on different arrays. So if one of them grew in size a bunch, it would have an effect on the run time.